src/Foundation.Cms/Pages/ExtendedStandardPage.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using EPiServer.Core;
+using EPiServer.DataAbstraction;
+using EPiServer.DataAnnotations;
+using EPiServer.Shell.ObjectEditing;
+using EPiServer.Web;
+using Foundation.Cms.Blocks;
+using Foundation.Cms.EditorDescriptors;
+
+namespace Foundation.Cms.Pages
+{
+    [ContentType(DisplayName = "Extended Standard Page",
+        GUID = "e6911472-0769-4092-a53c-7a6241df4dc3",
+        Description = "Allows for creation of rich extended standard pages",
+        GroupName = SystemTabNames.Content)]
+    [ImageUrl("~/assets/icons/cms/pages/CMS-icon-page-23.png")]
+    public class ExtendedStandardPage : FoundationPageData
+    {
+        [CultureSpecific]
+        [Editable(false)]
+        public override ContentArea MainContentArea { get; set; }
+
+        [CultureSpecific]
+        [Display(Name = "Left content area", GroupName = SystemTabNames.Content, Order = 201)]
+        public virtual ContentArea LeftContentArea { get; set; }
```

Comment:
For a future implementation it would be good to create an interface with `LeftContentArea` and `RightContentArea` properties to unify functionality